app/templates/apis/submit-your-app.html

```diff
@@ -10,22 +10,16 @@
 
   <h3>Power-Up Specific Details</h3>
 
-  <p>Our bar for 3rd party Power-Ups is that they should provide a user experience at the level of Trello itself. They should not do too much, nor too little. They should be free of bugs and awkward user experiences.</p>
+  <p>Our bar for Power-Ups is that they should provide a user experience at the level of Trello itself. They should not do too much, nor too little. They should be free of bugs and awkward user experiences.</p>
 
-  <p>We've put together a list of guidelines for building Power-Ups. Be sure to read through them <a href="power-ups/guidelines">here</a>.</p>
+  <p>Before you submitting your Power-Up please make sure you've done the following to make review as smooth as possible.</p>
```

Comment:
Think you combined two phrases at the start. Maybe it could be `Before you submit` or `Before submitting`